ceph.spec.in

```diff
@@ -211,18 +211,18 @@ BuildRequires:  protobuf-devel
 BuildRequires:  ragel
 BuildRequires:  systemtap-sdt-devel
 BuildRequires:  yaml-cpp-devel
-%if 0%{?fedora} || 0%{?rhel} >= 8
+%if 0%{?fedora}
 BuildRequires:  libubsan
 BuildRequires:  libasan
 BuildRequires:  libatomic
-%if 0%{with seastar}
+%endif
+%if 0%{?rhel}
```

Comment:
should we still have this conditional on seastar as well?

Reply:
it's already guarded by `%if 0%{with seastar}` at line 204.

Reply:
got it, was confused by the extra nested one